### PR TITLE
feat(apm): rework all IA

### PR DESCRIPTION
**Pull request by bradleycamacho**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/nav/apm.yml [hunk 1]
```diff
@@ -1,70 +1,13 @@
 title: APM agents
 path: /docs/apm
 pages:
-  - title: Getting started
+  - title: Get started with APM
```
Comment (bradleycamacho) [inline src/nav/apm.yml:4]: Not too sure about this title

File: src/nav/apm.yml [hunk 2]
```diff
@@ -1,70 +1,13 @@
...
-                path: /docs/apm/agents/manage-apm-agents/configuration/support-arm-graviton-x86-64
-          - title: Troubleshooting
-            path: /docs/apm/agents/manage-apm-agents/troubleshooting
-            pages:
-              - title: Configuration mismatch
-                path: /docs/apm/agents/manage-apm-agents/troubleshooting/agent-nrintegrationerrors-appear-insights
-              - title: Environment and services
-                path: /docs/apm/agents/manage-apm-agents/troubleshooting/get-environment-data-about-your-apm-app
-      - title: C SDK
+      - title: C SDK monitoring
```
Comment (bradleycamacho) [inline src/nav/apm.yml:10]: Is this the correct phrasing? Can you monitor an SDK?

File: src/nav/apm.yml [hunk 3]
```diff
@@ -1321,6 +1317,8 @@ pages:
             path: /docs/apm/applications-menu/features/configure-request-queue-reporting
           - title: Request queue config examples
             path: /docs/apm/applications-menu/features/request-queue-server-configuration-examples
+          - title: Alert information in APM
```
Comment (bradleycamacho) [inline src/nav/apm.yml:1320]: @paperclypse I hid this pending a larger IA revamp of APM, but let me know if you think this should go higher.

File: src/nav/apm.yml (hunk 2 above)
Comment (barbnewrelic) [reply src/nav/apm.yml:10]: C SDK is EOL so I wouldn't worry about the wording here

File: src/nav/apm.yml (hunk 3 above)
Comment (paperclypse) [reply src/nav/apm.yml:1320]: I think that's fine.

File: src/nav/apm.yml (hunk 1 above)
Comment (paperclypse) [reply src/nav/apm.yml:4]: I'll leave some suggestions on CodeStream.

Comment (paperclypse) [inline src/nav/apm.yml:4]: @bradleycamacho: Austin made a great point about focusing on what problem someone is trying to solve. There's a way in which the "APM" acronym doesn't serve us very well. It hides all the good words "application" "performance" "monitoring"! :D

Here are some alternatives (you may be able to swap "app" for "application" for brevity):
* Monitor your application's performance
* Start monitoring your application's performance
* Get started monitoring your application's performance
* Learn how to monitor your application
* Monitor your application

Thinking outside the box a little:
* Meet APM
* Choose your monitor
* Choose your application monitor
* Monitor and fine-tune your performance
* Start monitoring your app
* Why APM matters

**Review by paperclypse — DISMISSED** [review]
I think this is a good start!

**Comment by barbnewrelic** [comment]
@bradleycamacho I saw this PR in the Needs review column but it looks like @paperclypse  has already reviewed. I can review if you need another set of eyes on this one. Otherwise I'd say go ahead and move it along!

**Pull request merged**